README.md

```diff
@@ -1,118 +1,153 @@
-1. Use Homebrew to install the latest stable version of Ruby:
+To check Ruby version on your environment, run in your terminal:
 
-   ```
-   $ brew install ruby
-   ```
+```shell
+$ ruby -v
+ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-darwin18]
+```
 
-**Unix, Windows and other OS users**
+See the [Ruby website](https://www.ruby-lang.org/en/documentation/installation) for installation instructions.
 
-See the [Ruby site](https://www.ruby-lang.org/en/documentation/installation) for instructions.
+*Windows installation using package manager*:  See [Build DevDocs in Windows](#build-devdocs-in-windows)
 
-### Install Bundler
+#### Packaging system
 
-Install the [Bundler](https://bundler.io/) gem, which helps with Ruby dependencies:
+RubyGems is a packaging system in Ruby.
 
-```
-$ gem install bundler
-```
+Update the RubyGems:
 
-Once you have completed preparing your environment, you can build locally and review the site in your browser.
+```shell
+gem update --system
+```
 
-### Install devdocs
+### Installation
 
-Clone or download the repository. The first time you are at the `devdocs` directory, run:
+1. Clone the repository.
+2. Install the project dependencies using Bundler:
 
-```
+```shell
 $ bundle install
+<truncated output>
+Using parallel 1.12.1
+Using yell 2.0.7
+Using html-proofer 3.9.3
+Using progressbar 1.10.0
+Using verbal_expressions 0.1.5
+Using jekyll-algolia 1.4.8
+Using launchy 2.4.3
+Bundle complete! 5 Gemfile dependencies, 99 gems now installed.
+Use `bundle info [gemname]` to see where a bundled gem is installed.
 ```
 
+>**Note**
+>[Bundler](https://bundler.io/) is a tool to manage installation dependencies.
+>It is installed as a part of the [packaging system update](#packaging-system).
+
 Once you have completed preparing your environment, you can build locally and review the site in your browser.
 
-### To build locally:
+### Build locally
 
-#### Using rake
+#### Build using rake
 
-[rake](https://github.com/ruby/rake) is a native Ruby tool that helps to automate tasks.
+[rake](https://github.com/ruby/rake) is a Ruby tool that helps to automate tasks.
 
-1. Run the rake task that installs all required dependencies and starts the [Jekyll](https://jekyllrb.com/) server:
+Run the rake task that checks required dependencies and starts the Jekyll server:
 
-   ```
-   $ rake preview
-   ```
+```shell
+$ rake preview
+Install gems listed in the Gemfile: $ bundle install
+Using concurrent-ruby 1.1.3
+Using i18n 0.9.5
+Using minitest 5.11.3
+Using thread_safe 0.3.6
+Using tzinfo 1.2.5
+Using activesupport 4.2.10
+
+<truncated output>
+
+                    done in 14.865 seconds.
+Auto-regeneration: enabled for '/Users/user/Projects/repos/magento/devdocs'
+LiveReload address: http://127.0.0.1:35729
+    Server address: http://127.0.0.1:4000/
+Server running... press ctrl-c to stop.
+        LiveReload: Browser connected
+```
 
-1. Press `Ctrl+C` in the serve terminal to stop the server.
+This command:
```

Comment:
This is a strange way to introduce the following. It would be better to say 
"The 'rake' task performs the following actions:"
Or something similar to remind them what command you are describing.